### PR TITLE
Updated AssemblyStrongNameRule to verify public key tokens

### PR DESCRIPTION
**Pull request by ajaybhargavb**
https://github.com/aspnet/Universe/issues/310

@Eilon @pranavkm

### CODE REVIEW

File: src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs [hunk 1]
```diff
@@ -13,6 +15,8 @@ namespace NuGetPackageVerifier.Rules
 {
     public class AssemblyStrongNameRule : IPackageVerifierRule
     {
+        private static byte[] _publicKeyTokenBytes = new byte[] { 173, 185, 121, 56, 41, 221, 174, 96 };
```
Comment (Eilon) [inline src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs]: Actually can this be in hex? I think that would look more familiar to someone seeing it.

File: src/NuGetPackageVerifier/PackageIssueFactory.cs [hunk 2]
```diff
@@ -95,6 +95,11 @@ public static PackageVerifierIssue AssemblyNotStrongNameSigned(string assemblyPa
             return new PackageVerifierIssue("SIGN_STRONGNAME", assemblyPath, string.Format("The managed assembly '{0}' in this package is either not signed or is delay signed. HRESULT=0x{1:X}", assemblyPath, hResult), MyPackageIssueLevel.Error);
         }
 
+        public static PackageVerifierIssue AssemblyHasWrongPublicKeyToken(string assemblyPath)
+        {
+            return new PackageVerifierIssue("WRONG_PUBLICKEYTOKEN", assemblyPath, string.Format("The managed assembly '{0}' in this package does not have the correct public key token.", assemblyPath), MyPackageIssueLevel.Error);
```
Comment (Eilon) [inline src/NuGetPackageVerifier/PackageIssueFactory.cs]: Can the message show the public key token (i.e. `this assembly doesn't use... the b70f54315478 key`) or whatever?

**Comment by ajaybhargavb** [comment]
:up: :date:

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**